internal/compiler/widgets/fluent-base/std-widgets-base.slint

```diff
@@ -16,12 +16,12 @@ export { ComboBox }
 import { GroupBox } from "groupbox.slint";
 export { GroupBox }
 
+import { ListView, StandardListView } from "../common/listview.slint";
```

Comment:
Sorry to be nitpicking again, but I think this should stay below the `LineEdit`. :-)